livechat/configuration/client.py

```diff
@@ -490,6 +491,8 @@ def create_bot(self,
                 webhooks (dict): Webhooks sent to the Bot.
                 work_scheduler (dict): Work scheduler options to set for the new Bot.
                 timezone (str): The time zone in which the Bot's work scheduler should operate.
+                owner_client_id (str): Required only when authorizing via PATs.
+                The request will be rejected if you provide owner_client_id when authorizing with a Bearer Token.
```

Comment:
Please align second line like in payload description

Reply:
```suggestion
                                The request will be rejected if you provide owner_client_id when authorizing with a Bearer Token.
```